### ISSUE TITLE
Prevent esc (escape key) from showing meeting controls in Zoom

### ISSUE THREAD

**Issue by rattrayalex** [pqrs-org/Karabiner-Elements#3354]
Right now, in Zoom while screen-sharing, you can press ctrl-option-cmd-H to hide floating meeting controls. However, when you press "escape" (within any application), it shows the meeting controls again and focuses the Zoom application, preventing any further keystrokes from working in the application you were in. This is tremendously frustrating.

Related Zoom issue thread: https://community.zoom.com/t5/Meetings/How-can-I-disable-Escape-key-binding-in-Zoom/m-p/22247

Is there a way to disable this behavior with Karabiner-Elements?

**Comment by CHPrado** [comment]
Yes please! At least give us the option to disable ESC key shortcut

**Comment by henrebotha** [comment]
```json
{
    "description": "Block Esc key in Zoom",
    "manipulators": [
        {
            "conditions": [
                {
                    "bundle_identifiers": [
                        "us.zoom.xos"
                    ],
                    "type": "frontmost_application_if"
                }
            ],
            "from": { "key_code": "escape" },
            "to": [{ "key_code": "vk_none" }],
            "type": "basic"
        }
    ]
}
```

**Comment by henrebotha** [comment]
> {
>     "description": "Block Esc key in Zoom",
>     "manipulators": [
>         {
>             "conditions": [
>                 {
>                     "bundle_identifiers": [
>                         "us.zoom.xos"
>                     ],
>                     "type": "frontmost_application_if"
>                 }
>             ],
>             "from": { "key_code": "escape" },
>             "to": [{ "key_code": "vk_none" }],
>             "type": "basic"
>         }
>     ]
> }

Never mind, this works everywhere _except_ for showing the floating controls. 🙃

**Comment by henrebotha** [comment]
And it's obvious why.

```json
"type": "frontmost_application_if"
```

🙄 

What I really want is, "Replace <kbd>Esc</kbd> with `vk_none` where Zoom is concerned." I'm not convinced KE can do this.
